scimma_admin/hopskotch_auth/views.py

```diff
@@ -106,8 +124,12 @@ def delete(request):
 @login_required
 def download(request):
     myfile = StringIO()
-    myfile.write("username,password\n")
-    myfile.write(f"{request.POST['username']},{request.POST['password']}")       
+    myfile.write("username,password,kafkahost,nickname\n")
```

Comment:
`kafkahost` appears to be unused

Reply:
Oops, yes, that's part of something that I realized after I started writing it should be a subsequent, separate feature.